File: 4. Median of Two Sorted Arrays/MedianOfTwoSortedArrays.cs
public class Solution {
    public double FindMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] combinedList = nums1.Concat(nums2).ToArray();
        combinedList = combinedList.OrderBy(x => x).ToArray();
        
        if (combinedList.Length % 2 == 0) {
            int i = combinedList.Length / 2;
            return (combinedList[i-1] + combinedList[i]) / 2.0;
        } else {
            int i = combinedList.Length / 2;
            return combinedList[i];
        }
    }
}

File: 4. Median of Two Sorted Arrays/median_of_two_sorted_arrays.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        combined_list = nums1 + nums2
        combined_list.sort()
        
        if len(combined_list) % 2 == 0:
            i = len(combined_list) // 2
            result = (combined_list[i-1] + combined_list[i]) / 2.0
            return result
        else:
            i = len(combined_list) // 2
            return combined_list[i]